e2e-pw/src/oss/poms/saved-views.ts

```diff
@@ -0,0 +1,406 @@
+import { expect, Locator, Page } from "src/oss/fixtures";
+
+const ColorList = [
+  "Gray",
+  "Blue",
+  "Purple",
+  "Red",
+  "Yellow",
+  "Green",
+  "Pink",
+  "Orange",
+  "Purple",
+];
+type Color =
+  | "Gray"
+  | "Blue"
+  | "Purple"
+  | "Red"
+  | "Yellow"
+  | "Green"
+  | "Pink"
+  | "Orange"
+  | "Purple";
+
+const updatedView = {
+  name: "test updated",
+  description: "test updated",
+  color: "Yellow" as Color,
+};
+
+export const updatedView2 = {
+  name: "test updated 2",
+  description: "test updated 2",
+  color: "Orange" as Color,
+};
+
+export class SavedViewsPom {
+  readonly page: Page;
+  readonly assert: SavedViewAsserter;
+
+  readonly locator: Locator;
+  readonly dialogLocator: Locator;
+
+  savedViewNameA = "test name 1";
+
+  constructor(page: Page) {
+    this.page = page;
+    this.assert = new SavedViewAsserter(this);
+
+    this.locator = page.getByTestId("saved-views-select-container");
+    this.dialogLocator = page.getByTestId("saved-views-modal-container");
+  }
+
+  clearSavedViewsStmt() {
+    return `
+      import fiftyone as fo
+      dataset = fo.load_dataset("${this.savedViewNameA}")
+      dataset.delete_saved_views()
+    `;
+  }
+
+  async clickEditRaw(name: string = "test") {
```

Comment:
If it's the same name everywhere, I'd instead take `name` as a parameter in the constructor and refer to it as `this.name`. If not, I'd omit the default value. The default value for this arg has nothing to do with what a saved view is and therefore doesn't belong here.